index.css

```diff
@@ -1,23 +1,23 @@
-.panel#static {
+#static {
     position: fixed;
     left: 0;
     top: 0;
     height: 100%;
     overflow: hidden;
 }
 
-.panel#dynamic {
+#scrolling {
     height: 100%;
     margin-left: 50%;
 }
 
-.panel#static #anchor_img {
+#anchor_img {
     width: 100%;
     height: auto;
 }
 
-.panel#dynamic #main_content {
-    padding: 15px 35px 15px;
+#main_content {
+    padding: 0em 2em 0em;
```

Comment:
The changes to use relative measurements are good, but I think the content is a little cramped now. There is not much negative space on the top of the scrolling panel or on the left and right. Obviously this is subjective, so let me know what you think.